src/app/storage.js

```diff
@@ -2,7 +2,56 @@
 
 var utils = require('./utils');
 
-function StorageHandler (updateFiles) {
+function Storage (updateFiles) {
+  var EDITOR_SIZE_CACHE_KEY = 'editor-size-cache';
+
+  this.rename = function (originalName, newName) {
+    var content = this.get(originalName);
+    this.set(newName, content);
+    this.remove(originalName);
+  };
+
+  this.remove = function (name) {
```

Comment:
No particular reason. This approach works, and it's the approach I've used in all of my recent refactorings. If you'd like to rework those to use prototypes, create an issue an I'll investigate.